package.json

```diff
@@ -22,25 +22,25 @@
   "dependencies": {
     "@types/dotenv": "^8.2.0",
     "discord.js": "^13.1.0",
-    "dotenv": "^10.0.0",
-    "generic-type-guard": "^3.4.1",
+    "dotenv": "^16.0.3",
+    "generic-type-guard": "^3.7.0",
     "require-all": "^3.0.0",
     "signale": "^1.4.0",
-    "source-map-support": "^0.5.19"
+    "source-map-support": "^0.5.21"
   },
   "engine": {
     "node": ">=16.6.0",
     "npm": ">=7.0.0"
   },
   "devDependencies": {
-    "@types/node": "^16.4.13",
+    "@types/node": "^18.8.2",
     "@types/require-all": "^3.0.3",
-    "@types/signale": "^1.4.2",
-    "@typescript-eslint/eslint-plugin": "^4.29.2",
-    "@typescript-eslint/parser": "^4.29.2",
-    "eslint": "^7.32.0",
+    "@types/signale": "^1.4.4",
+    "@typescript-eslint/eslint-plugin": "^5.39.0",
+    "@typescript-eslint/parser": "^5.39.0",
+    "eslint": "^8.24.0",
     "eslint-config-google": "^0.14.0",
-    "tsc-watch": "^4.4.0",
-    "typescript": "^4.3.5"
+    "tsc-watch": "^5.0.3",
+    "typescript": "^4.8.4"
```

Comment:
Should use the ~ modifier here as typescript doesn't follow semver

Reply:
verdade, as vezes esqueço que trocou pra ts